daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -26,6 +26,7 @@ import (
 	"sync"
 	"unsafe"
 
+	"github.com/containerd/containerd/sys"
```

Comment:
Could you do another rebase, and update this to use `"github.com/containerd/containerd/pkg/userns"` ?

```suggestion
	"github.com/containerd/containerd/pkg/userns"
```


Full diff (with my other suggestion);

```patch
diff --git a/daemon/graphdriver/btrfs/btrfs.go b/daemon/graphdriver/btrfs/btrfs.go
index 677dc7aed4..0d0ee3a207 100644
--- a/daemon/graphdriver/btrfs/btrfs.go
+++ b/daemon/graphdriver/btrfs/btrfs.go
@@ -26,7 +26,7 @@ import (
        "sync"
        "unsafe"

-       "github.com/containerd/containerd/sys"
+       "github.com/containerd/containerd/pkg/userns"
        "github.com/docker/docker/daemon/graphdriver"
        "github.com/docker/docker/pkg/containerfs"
        "github.com/docker/docker/pkg/idtools"
@@ -601,8 +601,8 @@ func (d *Driver) Remove(id string) error {

        if err := subvolDelete(d.subvolumesDir(), id, d.quotaEnabled); err != nil {
                if d.quotaEnabled {
-                       if sys.RunningInUserNS() && strings.Contains(err.Error(), "operation not permitted") {
-                               err = errors.Wrap(err, "failed to delete subvolume without root (hint: remount btrfs on \"user_subvol_rm_allowed\" option, or update the kernel to >= 4.18, or change the storage driver to \"fuse-overlayfs\")")
+                       if userns.RunningInUserNS() && strings.Contains(err.Error(), "operation not permitted") {
+                               err = errors.Wrap(err, `failed to delete subvolume without root (hint: remount btrfs on "user_subvol_rm_allowed" option, or update the kernel to >= 4.18, or change the storage driver to "fuse-overlayfs")`)
                        }
                        return err
                }
```

Reply:
done